charts/canvas-oda/values.yaml

```diff
@@ -5,6 +5,16 @@
   # This is a YAML-formatted file.
   # Declare variables to be passed into your templates.
 global:
+  image:
+    # docker registry to pull images from
```

Comment:
Please add an example, which format is expected. E.g.

`# docker registry to pulll images from, e.g. "myreg.io/"`

Should the registry end with a "/" or not?